features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js

```diff
@@ -35,26 +35,24 @@ var operationModule = function () {
             var featureList = [];
             var feature;
             for (var i = 0; i < features.size(); i++) {
+                if (features.get(i).getType() != type){
+                    continue;
+                }
                 feature = {};
-                feature["id"] = features.get(i).getId();
-                feature["code"] = features.get(i).getCode();
-                feature["name"] = features.get(i).getName();
-                feature["description"] = features.get(i).getDescription();
-                feature["deviceType"] = features.get(i).getDeviceType();
-                feature["metadataEntries"] = [];
+                feature["operation"] = new String(features.get(i).getCode());
+                feature["name"] = new String(features.get(i).getName());
+                feature["description"] = new String(features.get(i).getDescription());
+                feature["deviceType"] = new String(features.get(i).getDeviceType());
+                feature["params"] = [];
                 var metaData = features.get(i).getMetadataEntries();
                 if (metaData && metaData != null) {
```

Comment:
Fixed in commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1